### PR TITLE
Screenshots update for platform customization

### PR DESCRIPTION
**Pull request by darluk**

### CODE REVIEW

File: content/users-guide/enterprise-tenant-bundle/customization.md [hunk 1]
```diff
@@ -12,8 +12,6 @@ Apart from various [configuration](#configuration) settings, you can use your in
 
 Click **{{< enterprise-tenant >}}** in the **Settings** menu to access these settings.
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/customization.md:13]: I would actually leave one initial screenshot at the beginning to show where the settings can be defined. I think, the the current one is not correct though, as it shows the Branding tab and not the Customization tab?

File: content/users-guide/enterprise-tenant-bundle/customization.md [hunk 2]
```diff
@@ -287,9 +271,6 @@ We highly discourage any use of alternative DNS configurations for the following
 Once the DNS configuration is in place and if a certificate with the given requirements is available, it can be easily
 uploaded to the platform.
 
-<img src="/images/users-guide/enterprise-tenant/et-domain-name.png" alt="Domain name">
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/customization.md:290]: Please extend the text and explain where and what to do, which is not clear without the screenshot. Something like:
"In the **Domain** tab under **Enterprise Tenant**, click **Upload certificate**." Not sure what happens then, but probably you have to select it from the file system?

File: content/users-guide/enterprise-tenant-bundle/customization.md [hunk 3]
```diff
@@ -287,9 +271,6 @@ We highly discourage any use of alternative DNS configurations for the following
...
-
-
 Afterwards, you can activate the domain with a single click. After the domain has been activated, you will be redirected
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/customization.md:274]: with a single click > Please be more precise. What do I have to click?

**Comment by BeateRixen** [comment]
@darluk This PR includes again the images for user hierarchy, which are already covered here: https://github.com/SoftwareAG/c8y-docs/pull/832?

**Comment by darluk** [comment]
> @darluk This PR includes again the images for user hierarchy, which are already covered here: #832?

yes, that commit is to be removed, will do it in a while together with other updates you recommend

File: content/users-guide/enterprise-tenant-bundle/customization.md (hunk 1 above)
Comment (darluk) [reply content/users-guide/enterprise-tenant-bundle/customization.md:13]: there is no customization tab, there are only Branding, Domain name and Configuration, for Branidng there is a screen below

File: content/users-guide/enterprise-tenant-bundle/customization.md (hunk 2 above)
Comment (darluk) [reply content/users-guide/enterprise-tenant-bundle/customization.md:290]: updated

**Comment by darluk** [comment]
closing this PR, as there is another branch created with  committed changes for ET platform customization changes only https://github.com/SoftwareAG/c8y-docs/pull/842, comments are applied

**Pull request closed without merging**